Src/ElasticScale.Client/ElasticScale.Common/InternalHelpers/ApplicationNameHelper.cs

```diff
@@ -1,9 +1,13 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+#if NETFRAMEWORK
+    using System.Data.SqlClient;
```

Comment:
why referencing System.Data.SqlClient? SDS is in servicing mode and is not getting any updates. Microsoft.Data.SqlClient is current ADO library and is gettting updates all on regular basis and it also supports .Net framwork. Couple of serious issues which still exists in SDS were addressed and fixed in MDS.